crates/sui-default-config/src/lib.rs

```diff
@@ -91,49 +83,6 @@ pub fn DefaultConfig(_attr: TokenStream, input: TokenStream) -> TokenStream {
     })
 }
 
-/// Return a set of derives that should be added to the struct to make sure it derives all the
-/// things we expect from a config, namely `Serialize`, `Deserialize`, and `Debug`.
-///
-/// We cannot add core derives unconditionally, because they will conflict with existing ones.
-fn core_derives(attrs: &[Attribute]) -> BTreeSet<Ident> {
-    let mut derives = BTreeSet::from_iter([
-        format_ident!("Serialize"),
-        format_ident!("Deserialize"),
-        format_ident!("Debug"),
-        format_ident!("Clone"),
-        format_ident!("Eq"),
-        format_ident!("PartialEq"),
-    ]);
```

Comment:
Very much in favor of removing this. Its always very very confusing to me when one Derive macro applies derives for other traits when derives for those traits already exist.